generators/messaging-adapter-generator/src/main/java/uk/gov/justice/raml/jms/core/MediaTypes.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.justice.raml.jms.core;
+
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+
+import java.util.Map;
+import java.util.stream.Stream;
+
+import org.raml.model.Action;
+import org.raml.model.ActionType;
+import org.raml.model.MimeType;
+
+class MediaTypes {
```

Comment:
Is it possible to give this a more meaningful name, as it doesn't really contain MediaTypes.